app/response_processor.py

```diff
@@ -47,7 +47,7 @@ def service_name(self, url=None):
             elif 'validate' in parts:
                 return 'SDX-VALIDATE'
         except AttributeError as e:
-            self.logger.error("No valid service name", exception=e)
+            self.logger.error("No valid service name", exception=str(e))
```

Comment:
Better to pass `exc_info` https://docs.python.org/3.6/library/logging.html#logging.debug. i.e. `self.logger.error("No valid service name", exc_info=e)`. Unless I've missed something and this is a special logger?

Reply:
I'm glad you mentioned that actually.  Looking at it now, it should just be logger.exception("...") as that adds the exception information in there for you.
We've done in in other places and it's worked, so we might as well do it here right?  No point in defining fields if we can get it for free instead!

Reply:
Yeah if you don't need to write a custom error message then probs best to just call `logger.exception`

Reply:
Oh, sure, if that log level is fine!